.kitchen.yml

```diff
@@ -20,14 +20,20 @@ platforms:
     image: centos-7-0-x64
 
 - name: ubuntu-12.04
+  run_list:
+    - recipe[apt]
```

Comment:
Ah, seems like it is not DO-specific. It is required for all drivers. Otherwise, TestKitchen is failed.
Please, skip my comment. :)